src/renderer/shared/ui-entities/TransactionDetails/TransactionDetails.tsx

```diff
@@ -2,29 +2,34 @@ import { type PropsWithChildren, memo, useMemo } from 'react';
 
 import { type Account, type Chain, type Wallet } from '@/shared/core';
 import { useI18n } from '@/shared/i18n';
+import { useToggle } from '@/shared/lib/hooks';
 import { DetailRow } from '@/shared/ui/DetailRow/DetailRow';
+import { Icon } from '@/shared/ui/Icon/Icon';
 import { Separator } from '@/shared/ui/Separator/Separator';
-import { FootnoteText } from '@/shared/ui/Typography/index';
+import { CaptionText, FootnoteText } from '@/shared/ui/Typography/index';
 import { Box } from '@/shared/ui-kit';
+import { AccountsModal } from '@/entities/staking';
 import { WalletIcon, walletUtils } from '@/entities/wallet';
 import { Account as AccountComponent } from '../Account/Account';
 import { AccountExplorers } from '../AccountExplorer/AccountExplorers';
 
 type Props = PropsWithChildren<{
   wallets: Wallet[];
   chain: Chain;
-  initiator: Account;
+  initiator: Account[];
   signatory?: Account;
   proxied?: Account;
 }>;
 
 export const TransactionDetails = memo(({ wallets, chain, proxied, initiator, signatory, children }: Props) => {
   const { t } = useI18n();
 
+  const [isAccountsOpen, toggleAccounts] = useToggle();
+
   const initiatorWallet = useMemo(() => {
     return (
       walletUtils.getWalletFilteredAccounts(wallets, {
```

Comment:
This functions already returns `null` in most recent `dev` so we can omit `?? null` in the end